packages/loader/container-loader/package.json

```diff
@@ -73,6 +73,7 @@
   },
   "devDependencies": {
     "@fluidframework/build-common": "^0.23.0",
+    "@fluidframework/container-runtime": "^0.49.0",
```

Comment:
Why is this dependency needed? This should break layer-check since container-loader shouldn't depend on container-runtime.

Reply:
Removing DeltaManager from the `container-loader` export causes the test file moved to fail. Specifically this line fails, when we made this move: `tests/src/test/containerRuntime.spec.ts → loader/src/test/containerRuntime.spec.ts` unless we import `container-runtime`.

```
import { ScheduleManager, DeltaScheduler } from "@fluidframework/container-runtime";
```
`npm run layer-check` still passes. 

We figured the sacrifice of moving the test to a less than optimal place was worth it.

Reply:
we need to find a different way to accomplish this. tests for the container-runtime should not live in the loader layer

Reply:
ideally the test can just use what is exported. if it really needs the deltamanger class i'd rather see the test just do a submodule import `import { DeltaManager } from "@fluidframework/container-loader/dist/deltaManager";`